apollo-router/tests/common.rs

```diff
@@ -538,6 +545,12 @@ impl IntegrationTest {
         let address = listener.local_addr().unwrap();
         let url = format!("http://{address}/");
 
+        let apollo_otlp_listener =
+            TcpListener::bind(SocketAddr::from(([127, 0, 0, 1], 0))).unwrap();
+        let apollo_otlp_address = apollo_otlp_listener.local_addr().unwrap();
+        // This should ideally use the process_endpoint function to add the /v1/metrics suffix.
```

Comment:
Can remove this now (we shouldn't change the endpoint here anyway so we can eventually support traces with the same server just a different path)
```suggestion
```